packages/fxa-auth-server/config/index.ts

```diff
@@ -804,6 +804,15 @@ const conf = convict({
       format: String,
       doc: 'A shared secret to authenticate Stripe webhook requests',
     },
+    taxIds: {
+      doc: 'Mapping of currency to tax ID to show on invoices.',
+      format: Object,
+      default: {
+        EUR: 'EU1234',
+        CHF: 'CH1234',
+      },
+      env: 'TAXIDS',
+    },
```

Comment:
Update the deploy doc or whatever needed so we don't forget the actual tax id config values for prod?